src/lib/OpenEXR/ImfStandardAttributes.cpp

```diff
@@ -68,34 +68,60 @@ using namespace std;
 
 OPENEXR_IMF_INTERNAL_NAMESPACE_SOURCE_ENTER
 
-   
+IMF_STD_ATTRIBUTE_IMP (acesImageContainerFlag, AcesImageContainerFlag, int)
+IMF_STD_ATTRIBUTE_IMP (originalImageFlag, OriginalImageFlag, int)
 IMF_STD_ATTRIBUTE_IMP (chromaticities, Chromaticities, Chromaticities)
 IMF_STD_ATTRIBUTE_IMP (whiteLuminance, WhiteLuminance, float)
 IMF_STD_ATTRIBUTE_IMP (adoptedNeutral, AdoptedNeutral, V2f)
-IMF_STD_ATTRIBUTE_IMP (renderingTransform, RenderingTransform, string)
-IMF_STD_ATTRIBUTE_IMP (lookModTransform, LookModTransform, string)
-IMF_STD_ATTRIBUTE_IMP (xDensity, XDensity, float)
 IMF_STD_ATTRIBUTE_IMP (owner, Owner, string)
+IMF_STD_ATTRIBUTE_IMP (creator, Creator, std::string)
```

Comment:
what does it mean that some of these are "string" and others are "std::string"?

Reply:
to the compiler, it's nothing ~ we have hoisted std ~ I suggest that `std::string` is more readable than the extant `string` because `std::string` does not prompt questions about what sort of `string` we're looking it. We should do a little cleanup to `std::string` in these attribute declarations...

Reply:
Cary removed the std ~ it looks clean now